### DISCUSSION TITLE
Make draggable elements, so that they can't overlap

### DISCUSSION THREAD

**Discussion by Sagar1711** [clauderic/dnd-kit#689] [Q&A]
Hi,
I am building a layout system with draggable elements, demo link https://react-zxaigj.stackblitz.io
In one area I need multiple draggable elements such that if they overlap each other either swap with each other's position or return to their previous position.
![Capture](https://user-images.githubusercontent.com/19650838/161615508-ce2ed389-5840-442e-999c-b1c38dbe7631.PNG)

@clauderic Please let me know any suggestions
Thanks a lot

**Comment by Sagar1711** [comment]
@clauderic Any suggestions?
Please let me know
Thanks

**Reply by Sagar1711** [reply]
Can anyone please help me with this?

**Reply by sebastianservat** [reply]
Can you please create a CodeSandbox and share the link?
